testing/mod.ts

```diff
@@ -170,19 +169,35 @@ async function runTestsParallel(
 async function runTestsSerial(
   stats: TestStats,
   tests: TestDefinition[],
-  exitOnFail: boolean
+  exitOnFail: boolean,
+  disableLog: boolean
 ): Promise<void> {
   for (const { fn, name } of tests) {
     // See https://github.com/denoland/deno/pull/1452
     // about this usage of groupCollapsed
-    console.groupCollapsed(`test ${name} `);
+    let l, e;
+    if (disableLog) {
+      l = console.log;
+      e = console.error;
+      console.log = function() {};
+      console.error = function() {};
```

Comment:
This doesn't work when run in parallel.

Reply:
as i said:

> Currently it is set to false and only work for serialTesting. Do you think this is usefull and should i expand it to parallel? (needs a bit more refactor)

Reply:
How would you be able to do it in parallel? You're modifying global variables. I don't think this approach works.

Reply:
See updated commits. It works in parallel with and without logs.